common/v2/components/Dropdown.tsx

```diff
@@ -57,12 +67,6 @@ const DropdownContainer = styled('div')`
     padding: 16px 15px 16px 15px;
     position: inherit;
   }
-
-  .is-open > .Select-control > .Select-multi-value-wrapper > .Select-input:only-child {
```

Comment:
How come we don't need this anymore?

Reply:
This is the cause of downshifting because we need to downshift only when there is value in dropbox and not every time when the dropdown is open.